doc/classes/PropertyTweener.xml

```diff
@@ -59,7 +78,26 @@
 
 				func tween_curve(v):
 				    return curve.sample_baked(v)
-				[/codeblock]
+				[/gdscript]
+				[csharp]
+				[Export]
+				public Curve Curve { get; set;}
+
+				public override void _Ready()
+				{
+				    Tween tween = CreateTween();
+				    // Interpolate the value using a custom curve.
+				    Callable tweenCurveCallable = Callable.From((float value) =&gt; TweenCurve(value));
+				    tween.TweenProperty(this, "position:x", 300.0f, 1.0f).AsRelative().SetCustomInterpolator(tweenCurveCallable);
+				}
+
+				public TweenCurve(float value)
+				{
+				    return Curve.SampleBaked(value);
+				}
+				[/csharp]
+				[/codeblocks]
+
```

Comment:
```suggestion
```
This time it's just complaining about an extra empty line.

Reply:
I'm just gonna click the button again